clients/drcachesim/analyzer.cpp

```diff
@@ -98,6 +87,11 @@ get_reader(const std::string &path, int verbosity)
             return nullptr;
         }
         for (; iter != end; ++iter) {
+            const std::string fname = *iter;
+            if (fname == "." || fname == ".." ||
```

Comment:
Can't we use `starts_with(fname, ".")` and `starts_with(fname, "..")` here?

Reply:
We don't want to match names that start with dot though, like ".bashrc": this is just skipping the special `.` and `..` entries for the current and parent dir, so we want to match the entire string.